code-challenges.js

```diff
@@ -31,24 +35,65 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+const q1 = (temp) => {
+    if (temp === 212) {
+        return `${temp} is at boiling point.`
+    } else if (temp > 212) {
+        return `${temp} is above boiling point.`
+    } else if (temp < 212) {
+        return `${temp} is below boiling point.`
+    } else {
+        return `${temp} is not an accepted number.`
+    }
+}
+console.log(q1(temp1))  // output: "42 is below boiling point"
+console.log(q1(temp2))  // output: "350 is above boiling point."
+console.log(q1(temp3))  // output: "212 is at boiling point."
+console.log(q1("fish"))  // output: "fish is not an accepted       number."
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
 // Describe your process:
+    // create function named q2
+    // include parameters using two variables
+    // within function, create new variable Merged consisting of a combination of var1 and var2 via .concat accessor
+    // return total number of elements in new variable's array using .length built-in method
+    // invoking function should return total combined number of values from given arrays, or any other arrays used.
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
 // Expected output: 9
 
+const q2 = (var1, var2) => {
```

Comment:
Great work in making this a function!!  Keep in mind variable naming conventions.  Variables in javascript are written in camelCase.